internal/storage/blob/store.go

```diff
@@ -322,3 +322,7 @@ func (s *Store) ListSchemaIDs(ctx context.Context) ([]string, error) {
 func (s *Store) LoadSchema(ctx context.Context, url string) (io.ReadCloser, error) {
 	return s.idx.LoadSchema(ctx, url)
 }
+
+func (s *Store) LoadPolicy(ctx context.Context, file string) (*policy.Wrapper, error) {
```

Comment:
We can pass a list of files here instead of just one. For disk stores it might not make much difference whether we retrieve policies one by one or as a list. But, for network stores like blob or database, it's almost always much more efficient to retrieve a batch at once.